app/views/snippets/form_error_radio_show_errors.html

```diff
@@ -37,12 +37,12 @@ <h1 class="heading-large">
 
       </legend>
 
-      <label class="block-label" for="personal_details_yes">
+      <label class="block-label selection-button-radio" for="personal_details_yes">
         <input id="personal_details_yes" type="radio" name="personalDetails" value="Yes">
         <span></span> Yes, my personal details are correct
```

Comment:
Should these nested `<span>` elements also be removed? They have been removed in other examples.